catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/image/NitfPreStoragePlugin.java

```diff
@@ -207,9 +210,14 @@ private ContentItem createDerivedImage(String id, String qualifier, BufferedImag
         return null;
     }
 
-    private String buildDerivedImageTitle(String title, String qualifier) {
+    protected String buildDerivedImageTitle(String title, String qualifier) {
         String rootFileName = FilenameUtils.getBaseName(title);
-        return String.format(DERIVED_IMAGE_FILENAME_PATTERN, qualifier, rootFileName, JPG);
+
+        return (StringUtils.isNotBlank(rootFileName) &&
```

Comment:
This method should not actually check that the title is _only_ alphanumeric, but rather ensure that it _contains some_ alphanumeric values. Since this method strips out all chars that match [\W], underscores will remain. This check makes sure the filename does not contain only underscore, such that the resulting filename would be `{qualifier}-_.jpg`.